diff --git a/index.d.ts b/index.d.ts
index 5ac9a17a0eaf34623410f76aad544be30f621381..0531843b589c3cda813462ad822adb889c396ac4 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -2,12 +2,13 @@
 // Project: https://github.com/arthurbergmz/webpack-pwa-manifest
 // Definitions by: Arthur A. Bergamaschi <https://www.github.com/arthurbergmz>
 
-import { Plugin } from 'webpack';
+import { Compiler } from 'webpack';
 
-export = WebpackPwaManifest
+export = WebpackPwaManifest;
 
-declare class WebpackPwaManifest extends Plugin {
+declare class WebpackPwaManifest {
     constructor(options: WebpackPwaManifest.ManifestOptions);
+    apply: (compiler: Compiler) => void;
 }
 
 declare namespace WebpackPwaManifest {
